label_studio/projects/api.py

```diff
@@ -223,7 +227,7 @@ def patch(self, request, *args, **kwargs):
         # config changes can break view, so we need to reset them
         if label_config:
             try:
-                has_changes = config_essential_data_has_changed(label_config, project.label_config)
```

Comment:
hm, this variable looks like perhaps we should be doing something with it? I would be in favor of noqa-ing the error instead